README.md

```diff
@@ -102,7 +102,7 @@ Even though in some programming languages is common to name variables or object
     }
 
 ##### Reasoning
-If the client expects an array it could ( and would ) be confusing to receive a dictionary with one key containing the array as a value. 
+If you're calling a **GET** on `/entities` , you're asking for entities, not an object with the entities inside one of its properties. Think about it as a christmas gift. If you ask santa for all the power rangers action figures ( and i'm talking about the 90's ones ) and you only see one box under the tree, you can't know if you're getting just one or all of them wrapped in one gift. _Also,  ALL the power rangers? Couldn't you just ask for ONE toy?_
```

Comment:
Hahahaha 🤣  Although it made me laugh out loud, we should **not** include these kind of comments in our guidelines. Sorry, @agerace , but I encourage removing it.